### PR TITLE
Add 2017-01 Ruby meets Rust event to Website

### PR DESCRIPTION
**Pull request by killercup**
cf. #19

### CODE REVIEW

File: _posts/2017-01-18-ruby-meets-rust.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+location: coworking-cologne
+talks:
+- title: "Using Rust to make Ruby faster"
+  speaker:
+    name: "Jan-Erik"
+    github: badboy
+    twitter: badboy_
+---
+We are pleased to announce another cooperative meetup, this time with the folks of Cologne.rb, the local Ruby user group.
+So join us on the _third_ Wednesday in January 2017 at Railslove (Coworking Cologne) for an evening of everything that starts with "Ru"!
```
Comment (killercup) [inline _posts/2017-01-18-ruby-meets-rust.md:18]: Hopefully a great chance to add some puns throughout other communications :)

File: _posts/2017-01-18-ruby-meets-rust.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+title: "Ruby meets Rust"
+date: 2017-01-18 19:00:00 MEST
+categories: meetup cologne
+links:
+    "Meetup.com": "https://www.meetup.com/RustCologne/events/235877954/"
+    "Cologne.rb": "http://www.colognerb.de/events/januar-meetup-2017"
+    "Planning discussion on Github": "https://github.com/Rustaceans/rust-cologne/issues/19"
+location: coworking-cologne
+talks:
+- title: "Using Rust to make Ruby faster"
```
Comment (killercup) [inline _posts/2017-01-18-ruby-meets-rust.md]: @badboy I haven't seen a final title or description so I went with this. Feel free to edit.

File: _posts/2017-01-18-ruby-meets-rust.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  speaker:
+    name: "Jan-Erik"
+    github: badboy
+    twitter: badboy_
+---
+We are pleased to announce another cooperative meetup, this time with the folks of Cologne.rb, the local Ruby user group.
+So join us on the _third_ Wednesday in January 2017 at Railslove (Coworking Cologne) for an evening of everything that starts with "Ru"!
+
+PS: This meetup will likely be held in German, we will however reevaluate this at the beginning of the evening and may switch to English if needed.
```
Comment (killercup) [inline _posts/2017-01-18-ruby-meets-rust.md]: I went with our default here.

**Review by Florob — APPROVED** [review]

File: _posts/2017-01-18-ruby-meets-rust.md (hunk 2 above)
Comment (badboy) [reply _posts/2017-01-18-ruby-meets-rust.md]: I opted for "Rust and Ruby - make it fast".

**Review by badboy — APPROVED** [review]

**Comment by badboy** [comment]
With the title change applied, I approve of this PR!

**Comment by killercup** [comment]
Title changed. Thanks for not calling it "make it ~~great~~ fast again"!

File: _posts/2017-01-18-ruby-meets-rust.md (hunk 3 above)
Comment (colin-kiegel) [reply _posts/2017-01-18-ruby-meets-rust.md]: I suggest
`The language of this meetup is yet to be decided and will be announced soon (either German or English).`

**Review by colin-kiegel — APPROVED** [review]
I updated the save-the-date info over at meetup and suggest to publish that soon. :-)

https://www.meetup.com/de-DE/RustCologne/events/235877954/

**Comment by killercup** [comment]
Addressed all comments.

**Pull request merged**